### ISSUE TITLE
Preset Auth Token from User Session

### ISSUE THREAD

**Issue by mmaedler** [nelmio/NelmioApiDocBundle#1353]
Hi there,

our API-Documention currently requires a login before it can be reviewed and the sandbox used to try out the endpoints of our api.

Since the logged in state of the user implies that its api key is available through the auth-session, it would be nice to preset it so that the user can instantly start working.
My inital thought was to override the template, but that doesn't work since the template is not loaded from `MyBundle/Resources/NelmioApiDocBundle/views/layout.html.twig`. 

Is the above mentioned even possible? If so, how can I achieve it? 

Thanks!

PS: One addition: We are currently using Symfony 3.4 w/ ApiDocBundle 2.13.3

**Comment by GuilhemN** [comment]
We need to allow to customize our template indeed (contribution welcome). Meanwhile you can override `index.html.twig` (but you need to rewrite all of it), see http://symfony.com/doc/current/bundles/override.html#templates.

**Comment by mmaedler** [comment]
Hi @GuilhemN! Thanks for your reply. I thought that was what I was already doing. I followed this document (http://symfony.com/doc/3.4/templating/overriding.html) and placed a copy of `index.html.twig` into my projects bundle folder (`MyBundle/Resources/NelmioApiDocBundle/views/layout.html.twig`) and changed minor things (e.g. headlines), but changes did not appear even though I cleared my apps cache.

Is it possible that the version we are using atm (2.13.3) is not supporting template inheritance?

Thanks again!
